test/gtest/uct/ib/test_rc.cc

```diff
@@ -226,7 +244,30 @@ UCS_TEST_P(test_rc_iface_address, size_flush_remote)
     check_sizes(m_entity_flush_rkey, sizes);
 }
 
-UCT_INSTANTIATE_RC_DC_GGA_TEST_CASE(test_rc_iface_address)
+UCS_TEST_SKIP_COND_P(test_rc_iface_flush_remote, put_fence_no_flush_remote,
+                     !check_caps(UCT_IFACE_FLAG_PUT_ZCOPY))
+{
+    mapped_buffer sendbuf(1024, 0ul, *m_e1);
+    mapped_buffer recvbuf(1024, 0ul, *m_e2);
```

Comment:
maybe use smaller buffer? not sure every transport will support 1k zcopy (or need to check max_put_zcopy)